CONTRIBUTING.md

```diff
@@ -2,6 +2,18 @@
 
 Contributions are always welcome, both reporting issues and submitting pull requests!
 
+### Setup
+
+Follow these steps to run the shopify_python tests:
+
+```bash
+pyenv local 3.6.1 # or whatever version you want to work with
```

Comment:
For brand new docs we should at least use the most up to date version `3.6.5`

Reply:
I've updated it to 3.6.3 since this is the specific version of 3.6 running on CI.